appdirs.py

```diff
@@ -41,6 +46,12 @@
     system = sys.platform
 
 
+def _effective_user():
```

Comment:
This function should probably be placed at the bottom of the file, because all the support functions are located there.